### PR TITLE
[16.0][MIG] l10n_jp_country_state: Migration to 16.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**

### CODE REVIEW

**Comment by yostashiro** [comment]
@AungKoKoLin1997 https://github.com/odoo/odoo/issues/93671#issuecomment-1162526527

**Review by yostashiro — COMMENTED** [review]
@AungKoKoLin1997 Can you please consider the use of `i18n_extra` folder.  I suppose the .po file won't be removed upon merge this way (haven't confirmed), unlike those in `i18n`.

File: l10n_jp_country_state/data/res_country_state.xml
Comment (yostashiro) [inline l10n_jp_country_state/data/res_country_state.xml:1]: I suppose we don't need this file, do we?

File: l10n_jp_country_state/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This module updates country state data of Japan with translation.
```
Comment (yostashiro) [inline l10n_jp_country_state/readme/DESCRIPTION.rst]: To reflect the current scope of the module.
```suggestion
This module only adds translations to country state records for Japan, for the sake of convenience.
```

**Review by yostashiro — APPROVED** [review]
Code review.

**Review by liuhehe1995 — APPROVED** [review]
LGTM and Runboat tests are working.
Please refer to the screenshot for details.
Before1
After1

**Pull request merged**